docs/guides/annotations.md

```diff
@@ -299,3 +299,53 @@ A value for the y-value of the upper range flag dot
     "sourceView": ["index.html", "data.csv"]
 }
 ```
+
+## Confidence Area
+
+The area is a shaded area that represents a confidence interval. It is used to show the range of values within which the true value is likely to fall.
+
+### sszvis.annotationConfArea
```

Comment:
I'd write this out, ie. `sszvis.annotationConfidenceArea`. The short form `Conf` may be ambigous or difficult to understand for someone not familar with the project.